CastleNathria/Locales/frFR.lua

```diff
@@ -23,28 +23,28 @@ if L then
 	-- L.level = "%s (Level |cffffff00%d|r)"
 	-- L.full = "%s (|cffff0000FULL|r)"
 
-	-- L.custom_off_experimental = "Enable experimental features"
-	-- L.custom_off_experimental_desc = "These features are |cffff0000not tested|r and could |cffff0000spam|r."
+	L.custom_off_experimental = "Activer les options expérimentales"
+	L.custom_off_experimental_desc = "Ces options |cffff0000ne sont pas testées|r et pourraient |cffff0000spam|r."
 
-	-- L.anima_tracking = "Anima Tracking |cffff0000(Experimental)|r"
-	-- L.anima_tracking_desc = "Messages and Bars to track anima levels in the containers.|n|cffaaff00Tip: You might want to disable the information box or bars, depending your preference."
+	L.anima_tracking = "Suivi de l'anima |cffff0000(Experimental)|r"
+	L.anima_tracking_desc = "Messages et Barres pour suivre le niveau d'anima dans les conteneurs.|n|cffaaff00Tip: Vous pouvez désactiver les barres et boites d'infos si besoin."
 
-	-- L.custom_on_stop_timers = "Always show ability bars"
-	-- L.custom_on_stop_timers_desc = "Just for testing right now"
+	L.custom_on_stop_timers = "Toujours montrer les barres de techniques"
+	L.custom_on_stop_timers_desc = "En cours de test"
 
-	-- L.bottles = "Bottles"
-	-- L.sins = "Sins"
+	L.bottles = "Bouteilles"
+	L.sins = "Péchés"
```

Comment:
I think it would be better to use "Vices" instead of "Péchés" to match the in-game translation.